Additions/UIView-KIFAdditions.m

```diff
@@ -571,14 +571,14 @@ - (void)dragAlongPathWithPoints:(CGPoint *)points count:(NSInteger)count;
     [self dragPointsAlongPaths:@[[array copy]]];
 }
 
-- (void)dragPointsAlongPaths:(NSArray *)arrayOfPaths {
-    // must have at least one path, and each path must have the same number of points
-    if (arrayOfPaths.count == 0)
+- (void)dragPointsAlongPaths:(NSArray<NSArray<NSValue *> *> *)arrayOfPaths {
```

Comment:
Should we also be making similar changes to `- (NSArray *)pointsFromStartPoint:(CGPoint)startPoint toPoint:(CGPoint)toPoint steps:(NSUInteger)stepCount`? It's internal to `UIView-KIFAdditions`, but it also returns an `NSArray *` - should that be returning an `NSArray<NSArray<NSValue *> *> *`?

Reply:
Done